eng/scripts/Language-Settings.ps1

```diff
@@ -583,7 +583,7 @@ function GetExistingPackageVersions ($PackageName, $GroupId=$null)
     return $null
   }
   catch {
```

Comment:
Does this one not return 404 like the others?

Reply:
It does not error. I just returns an empty object.